src/main/java/com/aws/greengrass/ipc/IPCEventStreamService.java

```diff
@@ -75,6 +81,7 @@ public class IPCEventStreamService implements Startable, Closeable {
     @Override
     public void startup() {
         Path rootPath = kernel.getNucleusPaths().rootPath();
+        String ipcPath = Coerce.toString(deviceConfiguration.getIpcSocketPath());
```

Comment:
Convert the path into a `Path`

```suggestion
        String ipcPathStr = Coerce.toString(deviceConfiguration.getIpcSocketPath());
        Path ipcPath = Utils.isEmpty(ipcPathStr) ? null : Paths.get(ipcPathStr);
```